src/Forms/Email.php

```diff
@@ -132,10 +134,15 @@ protected function addAttachments()
     protected function addData()
     {
         $augmented = $this->submission->toAugmentedArray();
+        $fields = $this->getRenderableFieldData(Arr::except($augmented, ['id', 'date', 'form']));
+
+        if (array_has($this->config, 'attachments')) {
+            $fields = $fields->reject(fn ($field) => $field['fieldtype'] === 'assets');
```

Comment:
Why do asset fields need to get rejected?

Reply:
In fact, that's made me realise we should be filtering out Files fields here too.